website/content/api-docs/secret/ldap.mdx

```diff
@@ -181,6 +183,9 @@ The `static-role` endpoint configures Vault to manage the passwords of existing
 - `rotation_period` `(string: <required>)` - How often Vault should rotate the password of the user entry. Accepts
   [duration format strings](/vault/docs/concepts/duration-format). The minimum rotation period is 5 seconds.<br />
   **Example:** `"3600", "5s", "1h"`
+- `skip_import_rotation` `(boolean: <optional>)` - When set on a role creation request, Vault will not rotate the
```

Comment:
Should we explicitly state that this overrides the `skip_static_role_import_rotation` setting on the config?

Reply:
yeah, we should add language about the interaction here